LICENSE

```diff
@@ -200,9 +200,14 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-   This product bundles various third-party components also under the
-   Apache Software License 2.0.
+--------------------------------------------------------------------------------
 
-   Gradle (https://github.com/gradle/gradle)
-   ./gradlew
-   ./gradle/wrapper/gradle-wrapper.properties
\ No newline at end of file
+This product includes a gradle wrapper.
+
+* gradlew and gradle/wrapper/gradle-wrapper.properties
```

Comment:
Technically I'm not sure whether `gradle-wrapper.properties` actually counts. It's _generated_ by Gradle code, but modified by us.
OTOH, it probably doesn't hurt?